java/arcs/core/host/AbstractArcHost.kt

```diff
@@ -93,6 +97,21 @@ abstract class AbstractArcHost(
     override suspend fun lookupArcHostStatus(partition: Plan.Partition) =
         lookupOrCreateArcHostContext(partition.arcId).arcState
 
+    override suspend fun pause() {
+        paused = true
+        runningArcs.forEach { (arcId, context) ->
+            val partition = contextToPartition(arcId, context)
+            stopArc(partition)
```

Comment:
Note that if stopArc() throws an exception, it'll fail to pause all the arcs. This might be ok if the caller checks the exception and doesn't do the delete. Otherwise, you might want to put a try/catch around the stopArc() call

Reply:
Added a catch here, so we can still try to stop the other arcs. but I am not really sure how to handle it, except for logging the error.